src/app/models/percentile-indicator-query-opts.model.ts

```diff
@@ -0,0 +1,7 @@
+import { IndicatorQueryOpts } from './indicator-query-opts.model';
+
+export interface PercentileIndicatorQueryOpts extends IndicatorQueryOpts {
+    params: {
```

Comment:
Will need to list all the properties of the extended `params` object here after having rebased against develop, due to library updates